docs/admin_guide.md

```diff
@@ -961,7 +971,7 @@ With the following content:
 
 ```ini
 [program:messenger]
-command=php /var/www/mbin/bin/console messenger:consume async async_ap failed --time-limit=3600
+command=php /var/www/mbin/bin/console messenger:consume old async outbox deliver inbox resolve receive failed --time-limit=3600
 user=www-data
 numprocs=4
```

Comment:
What about increasing this number.. Could be a good idea right? I ask this, because again I don't want to change this configuration for admins. So maybe increasing this number to like 6 is maybe not a bad idea?

Reply:
true, 6 sounds like a useful number. We should then also adjust the docker config to run 6 replicas of the messenger container by default

Reply:
I changed it to 6 now. Docker currently still only uses 1 process it seems.

Reply:
So we either need to setup supervisord inside the docker image as well.. Or.. you could leverage Docker Swarm (deploy) feature, like this: 

https://github.com/pavelsur07/womjoy-store/blob/8d820cfedcc902c883f38819395ef43a588d6b7b/docker-compose-production-ru.yml#L351

Reply:
@nobodyatroot mentioned something about docker replicas a while ago, so no docker swarm (I think)

Reply:
ow my bad, As you can see in the link I posted above I do mean [replicas](https://docs.docker.com/compose/compose-file/deploy/#replicas), I thought it was a swarm thingy, but apparently not. Apparently in the normal docker compose spec 2.19.1.